<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab_03</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_draw_method">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Метод построения</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="draw_method">
          <item>
           <property name="text">
            <string>Брезенхем</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Средняя точка</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Библиотечная функция</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Каноническое уравнение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Параметрическое уравнение</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_line_color">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>Цвет линии</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="colorBox">
          <item>
           <property name="text">
            <string>Фиолетовый</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Черный</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Синий</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Зеленый</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Задний фон</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Кол-во</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="circle_num_box">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Шаг</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="circle_step_box">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Координаты центра</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="circle_center_x">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>350</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="circle_center_y">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>350</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Радиус  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="circle_radius">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="add_circle_btn">
        <property name="text">
         <string>Построить окружность</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Координаты центра</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ellipse_center_x">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>350</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ellipse_center_y">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>350</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Полуоси                    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ellipse_halfax_x">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ellipse_halfax_y">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>150</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="add_ellipse_btn">
        <property name="text">
         <string>Построить эллипс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clean_screen_btn">
        <property name="text">
         <string>Очистить экран</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="do_benchmarks_btn">
        <property name="text">
         <string>Сравнить алогоритмы</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
